R/staticimports.R

```diff
@@ -81,3 +81,35 @@ str_extract <- function(text, pattern) {
     )
   )
 }
+# Generated by staticimports; do not edit by hand.
+# ======================================================================
+# Imported from pkg:stringstatic
+# ======================================================================
+
+`%||%` <- function(lhs, rhs) {
+  if (length(lhs) == 0) return(rhs)
+  lhs
+}
```

Comment:
This could be a problem https://github.com/rstudio/gradethis/blob/c1e0c978078036a4a3341240be8377d3462c221c/R/gradethis-package.R#L10